src/interfaces/index.ts

```diff
@@ -0,0 +1,26 @@
+export interface calendarState {
```

Comment:
By standards, interface and class names should always use UpperCamelCase

Reply:
Thanks, I just update the interfaces names in order to follow the standards.